apps/system/test/unit/notifications_test.js

```diff
@@ -144,6 +145,12 @@ suite('system/NotificationScreen >', function() {
     };
     navigator.mozL10n = MockL10n;
 
+    isDocumentHidden = false;
+    Object.defineProperty(document, 'hidden', {
+      configurable: true,
+      get: () => isDocumentHidden
```

Comment:
actually I made this on purpose: the real one does not change anything on set, and really we should not let anybody to change it anywhere, it would be a bug. So I think it's better to not have a set.

Tell me what you think :)